src/Pages/Home/Home.js

```diff
@@ -16,30 +15,20 @@ export default function Home(props) {
 
     return (
         <>
-            <Hero
-                //TODO: figure out how to add/update featured ep
-                episode={podcastEpisodes[13]}
-                setSelectedEpisode={setSelectedEpisode}
-            />
-
-            <ListenOn
-                offsetValue={"-200px"}
-                googleLink={
-                    "https://podcasts.google.com/feed/aHR0cHM6Ly9ob3BwZWR1cGVhc3QuaGlwY2FzdC5jb20vcnNzL2hvcHBlZHVwZ2FtaW5nZWFzdC54bWw"
-                }
-                appleLink={
-                    "https://podcasts.apple.com/us/podcast/hopped-up-gaming-east/id797058638"
-                }
-                spotifyLink={
-                    "https://open.spotify.com/show/2dTRwdB4bHYsCI2CN9OV2m"
-                }
-            />
-
-            <LatestEpisodes
-                latestEpisodes={latestEpisodes}
-                setSelectedEpisode={setSelectedEpisode}
-            />
+            {podcastEpisodes.length > 0 && (
```

Comment:
I don't think we need this check anymore here or below because we are doing it in app before passing it in as a prop